scheduler/models.py

```diff
@@ -168,20 +207,36 @@ def interval_seconds(self):
         }
         return timedelta(**kwargs).total_seconds()
 
+    def _prevent_duplicate_runs(self):
```

Comment:
I'm pretty sure this resolves issue #30 
It sounds like the problem described is caused by there being remaining repeats with a scheduled time in the past. In my experience when this happens rq-scheduler starts the job right away essentially trying to catch up. The code below essentially steps forwards in time until there are no repeats left or it arrives at some new `scheduled_time` in the future that is the original `scheduled_time` + `interval`*`n` where `n` is the number of repeats it took to get there and subtracted from the repeats remaining.